apis/submariner/manifests/namespace.go

```diff
@@ -16,21 +16,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package broker
+package manifests
 
 import (
+	"context"
+
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 )
 
-const (
-	SubmarinerBrokerNamespace = "submariner-k8s-broker"
-)
+func CreateNewBrokerNamespace(clientset *kubernetes.Clientset, namespace string) (brokernamespace *v1.Namespace, err error) {
+	return clientset.CoreV1().Namespaces().Create(context.TODO(), NewBrokerNamespace(namespace), metav1.CreateOptions{})
+}
```

Comment:
This is simply a direct wrapper for the passed clientset,.  I don't think this function is really necessary and certainly  not as API to import from the add-on.